docs/changelog.md

```diff
@@ -13,7 +13,7 @@ permalink: /changelog/
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
 * **feature-top-sites**
-  * Added `isDefault` to the top site entity, which allows application to specify a default top site that is added by the application. This is called through `TopSiteStorage.addTopSite`.
+  * ⚠️ **This is a breaking change**: Added `isDefault` to the top site entity. Please verify that the latest schema is used if the feature is used. This new property allows application to specify a default top site that is added by the application. This is called through `TopSiteStorage.addTopSite`.
```

Comment:
I think we can improve the guidance, as the schema is something that consumers can't see, it is a underlying detail. Maybe we could suggest if they are using the feature, they have test their apps and if they find any errors related to the schema they have to upgrade to the latest version of AC

Reply:
Fixed.